examples/dynamics/oscillator.ipynb

```diff
@@ -154,4 +156,4 @@
    "metadata": {}
   }
  ]
-}
```

Comment:
@jgosmann: in the diffs is always where I see it. I can't find a way to just view notebooks, either.